profile/E-ARK-CSIP-v2-1-1.xml

```diff
@@ -0,0 +1,2920 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- 2021-01-08 Draft of E-ARK CSIP METS Profile 2.1.1 -->
+<METS_Profile xmlns="http://www.loc.gov/METS_Profile/v2"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:mets="http://www.loc.gov/METS/"
+    xmlns:csip="https://DILCIS.eu/XML/METS/CSIPExtensionMETS"
+    xmlns:xlink="http://www.w3.org/1999/xlink"
```

Comment:
We will need at least a namespace prefix declaration here for the new Schematron entries with the `iso` namespace, e.g. `xmlns:iso="http://purl.oclc.org/dsdl/schematron"`